README.md

```diff
@@ -461,7 +464,7 @@ jobs:
     steps:
       - uses: actions/checkout@v3
 
-      # This third party action allows you to download the cache artifact from different workflow runs
+      # This third-party Action allows you to download the cache artifact from different workflow runs
```

Comment:
Can users now use GitHub's official `actions/cache`? Newer versions may have fixed problems we had before?

- https://github.com/renovatebot/github-action/issues/827
- https://github.com/renovatebot/github-action/issues/646

Reply:
yes, but i newer used this action, so don't now how to write this section.

Reply:
I don't understand it enough either. We already have an issue about updating the instructions to use the official `actions/cache` GitHub Action.

I'm resolving this review comment, as I can't fix it up myself. :wink:

Reply:
See https://github.com/renovatebot/github-action/issues/646#issuecomment-2465543441 for an example.